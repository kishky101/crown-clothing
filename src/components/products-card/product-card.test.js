import renderer from 'react-test-renderer';
import ProductCard from './product-card.component';
import { MockStore } from '../../utils/test/setup';

test('creating product-card snapshot', () => {
    const productcard = renderer.create(
        <MockStore>
            <ProductCard product={{
                id: 6,
                name: 'Palm Tree Cap',
                imageUrl: 'https://i.ibb.co/rKBDvJX/palm-tree-cap.png',
                price: 14,
            }} />
        </MockStore>
    ).toJSON;
    expect(productcard).toMatchSnapshot();
})