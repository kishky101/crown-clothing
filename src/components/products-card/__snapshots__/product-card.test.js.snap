// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating product-card snapshot 1`] = `[Function]`;
